code/outlierfunction.py

```diff
@@ -0,0 +1,19 @@
+
+# find outliers based on DVARS and FD
+
+def outlier(file_name, save_name, bound):
+    # load data
+    data = np.loadtxt(file_name)
+    outlier = []
+    # set outlier values to 0
+    for i in data:
+        if i <= bound:
+            outlier.append(0)
+        else:
+            outlier.append(i)
+    # find outlier indices
+    outlier_indices = np.nonzero(outlier)
+    # write file to store outlier indices
```

Comment:
yeah something like that, whatever is more convenient to access if we are to do plotting/removal of outliers. I am not sure if a dictionary save/load properly from txt, try it out